code/game/jobs/job/church.dm

```diff
@@ -23,9 +23,11 @@
 	department_account_access = TRUE
 	outfit_type = /decl/hierarchy/outfit/job/church/chaplain
 
+	wl_config_heads = TRUE //Eclipse edit
+
 	stat_modifiers = list(
 		STAT_TGH = 10,
-		STAT_BIO = 15,
+		STAT_BIO = 35,
```

Comment:
I can't seem to find a reason as to why this role requires such a heavy boost in BIO. If it does, it shouldn't be 35, since it's 5 skill points off of the starting BIO of a fully trained medical doctor, which doesn't make too much sense.